### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by JonasKruckenberg** [JonasKruckenberg/imagetools#7] [General]
## 👋 Hey!

Most of the weight of a website is made up of images, but it doesn't have to! Using modern image formats and clever compression you can **reduce the size of your website dramatically**. 
This is why I created this plugin in the first place, to make it easy for developers to **transform images**, **generate srcsets** and **work on your site**; without having to go back and forth between photoshop and your editor.

With that said, **Welcome to the Discussion Section** ✨!

- Share ideas, cool projects and more
- Ask questions regarding the plugin, images, whatever

So everything that is not a bug report, feature request or similar should go here.

Have fun!

**Comment by yamiteru** [comment]
SvelteKit team might be considering using this library, see https://github.com/sveltejs/kit/issues/241

**Reply by JonasKruckenberg** [reply]
Oh awesome! And thanks for doing a bit of advertisement😄
I'll keep an eye out for their development plans!

You also may have noticed that I haven't been as active lately (had to spend some time preparing for my university exams) but I'll be back next week and hopefully ready to publish the rewrite for good 👍🏻

**Reply by yamiteru** [reply]
Yeah no problem bro. I had to stop the work on my portfolio for now as well so great timing 👌
